### PR TITLE
Improved TCP source mapping, and fw rules apply just to intercept ifaces

### PR DESCRIPTION
**Pull request by CaffeinatedWill**

### CODE REVIEW

File: nac_bypass_setup.sh [hunk 1]
```diff
@@ -313,8 +316,398 @@ ConnectionSetup() {
...
 
     ## Re-enabling traffic flow; monitor ports for lockout
-    $CMD_ARPTABLES -D OUTPUT -j DROP
-    $CMD_IPTABLES -D OUTPUT -j DROP
+    $CMD_ARPTABLES -D OUTPUT -o $SWINT -j DROP
+    $CMD_ARPTABLES -D OUTPUT -o $COMPINT -j DROP
+    $CMD_IPTABLES -D OUTPUT -o $COMPINT -j DROP
+    $CMD_IPTABLES -D OUTPUT -o $SWINT -j DROP
+
+    ## Housecleaning
```
Comment (0x6d69636b) [inline nac_bypass_setup.sh:324]: What happened here? This is a duplicate of the code?

**Review by 0x6d69636b — CHANGES_REQUESTED** [review]
Thank you very much for your work! Could you please check my comment, something does not seem right to me

File: nac_bypass_setup.sh (hunk 1 above)
Comment (CaffeinatedWill) [reply nac_bypass_setup.sh:324]: 241-244 are adding drop rules to IPTables & ARPTables, and 319-322 are deleting those drop rules from IPTables & ARPTables

Comment (0x6d69636b) [reply nac_bypass_setup.sh:324]: That I understand, but 323 and following, you're adding ~400 lines of code to the script

Comment (CaffeinatedWill) [reply nac_bypass_setup.sh:324]: Holy crap you're absolutely right, idk how I missed that - yeah I must have duped the file. I'll close this pull, recommit with the valid one, and re-submit the pull. Thanks for the spot check!

**Pull request closed without merging**